controller/workController.js

```diff
@@ -3,6 +3,7 @@
 
   workController.reveal = function() {
     $('#about').hide();
```

Comment:
this could be done better by adding a class to all these hide-able sections, the just calling `$(".className").hide()`